task/sast-snyk-check/0.1/sast-snyk-check.yaml

```diff
@@ -9,7 +9,8 @@ metadata:
   name: sast-snyk-check
 spec:
   description: >-
-    Static application security testing with Snyk.
+    Uses Snyk Code tool to perform Static Application Security Testing (SAST) on application's source code.
```

Comment:
Suggestion based on how we have structured the descriptions above: Scan source code for security vulnerabilities, including common issues such as SQL injection, cross-site scripting (XSS), and code injection attacks with Snyk, a Static Application Security Testing (SAST) tool.

Question: ....to scan source code.... Which source code? container?

Reply:
I like this phrasing, thanks.
I'd replace "Snyk" with "Snyk Code" though because Snyk Code is the name of the tool we're using, and Snyk is the name of the company which built it, so it's crucial.

> Question: ....to scan source code.... Which source code? container?

Snyk Code checks the source code of your application, refer to https://snyk.io/product/snyk-code/ to know about the tool.
Just FYI: please check [this](https://docs.snyk.io/scan-application-code/snyk-code/snyk-code-language-and-framework-support#language-support-with-snyk-code-ai-engine) to know about the supported languages by Snyk Code's AI Engine.

Reply:
Thank you!